assets/js/main.js

```diff
@@ -69,3 +69,14 @@ $(".show-hide").click(function() {
     $(this).removeClass("ms-icon icon-hide").addClass("ms-icon icon-view");
   }
 });
+
+  // languague
+  $('.language-selector .option').click(function(){
+    $(this).toggleClass('open');
+  });
+
+  $(window).click(function(e) {
+    if ($('.language-selector').has(e.target).length == 0 && !$('.language-selector').is(e.target)) {
+      $('.language-selector .option').removeClass('open');
+    }
+  });
```

Comment:
Could you rewrite these scripts in order to make them work in any place (aka WebApp) as the previous ones? probably @fcoronelmakingsense or @damianmuti could help you with that.

Reply:
`$("body").on('click', '.language-selector .option', function() {
  $('.language-selector .option').toggleClass('open');
});`

Reply:
`$(document).on('mouseup', function(e){
  var menu = $(".user-menu");
  var languageSelector = $(".language-selector");

  if (!menu.is(e.target) && menu.has(e.target).length === 0){
    menu.removeClass('open');
  }

  if (!languageSelector.is(e.target) && languageSelector.has(e.target).length === 0){
    $('.language-selector .option').removeClass('open');
  }
});`